tests/kafkatest/services/streams.py

```diff
@@ -52,6 +52,33 @@ class StreamsTestBaseService(KafkaPathResolverMixin, JmxMixin, Service):
         "streams_stderr": {
             "path": STDERR_FILE,
             "collect_default": True},
+        "streams_log.1": {
+            "path": LOG_FILE + ".1",
```

Comment:
will the system test results still get errors when these files aren't found?

Reply:
Yes, we need to specify all used files here -- otherwise they won't be collected after the test finished.